test/run

```diff
@@ -143,24 +161,29 @@ def print_counter(counter):
 
 
 def main():
+    args = get_args()
     logging.basicConfig(format="run-tests: %(message)s")
     test_dir = pathlib.Path(__file__).resolve().parent
     root = test_dir.parent
     litani = root / "litani"
+    if args.output_dir:
+        output_dir = pathlib.Path(args.output_dir)
+    else:
+        output_dir = test_dir / "output"
 
     run_cmd([
         litani, "init",
         "--project", "Litani Test Suite",
-        "--output-prefix", test_dir / "output",
-        "--output-symlink", test_dir / "output" / "latest"])
+        "--output-prefix", output_dir,
+        "--output-symlink", output_dir / "latest"])
 
     counter = {
         "added": 0,
         "total": 0,
     }
 
     add_unit_tests(litani, test_dir, root, counter)
-    add_e2e_tests(litani, test_dir, root, counter)
+    add_e2e_tests(litani, test_dir, root, counter, output_dir=output_dir)
```

Comment:
`output_dir` will always have a value due to L169-172. `output_dir` should be part of this function's `*args`.

Reply:
Resolved